src/esw/brushless.py

```diff
@@ -234,8 +335,44 @@ def __init__(self):
         else:
             transport = None
 
-        for name, info in drive_info_by_name.items():
-            self.drive_bridge_by_name[name] = MoteusBridge(info["id"], transport)
+        self._motors_manager = MotorsManager(arm_controller_info_by_name, transport, "arm_status")
+        rospy.Subscriber("ra_cmd", JointState, self._process_ra_cmd)
+
+    def _process_ra_cmd(self, ros_msg: JointState) -> None:
+        """
+        Process a JointState command controls individual motor speeds.
+        :param ros_msg: Has information to control the joints controlled by both brushed and brushless motors.
+        We only care about the joints controlled by brushless motors.
+        """
+        self._motors_manager.update_time()
+
+        for i, name in enumerate(ros_msg.name):
+            if name in self._arm_names:
+                position, velocity, torque = (
+                    ros_msg.position[i],
+                    ros_msg.velocity[i],
+                    ros_msg.effort[i],
+                )
+
+                # We usually assume that the ros_msg sends velocity commands from -1 to 1,
+                # but change it just in case.
+                if abs(velocity) > 1:
```

Comment:
Taking a second pass at this. Should we be worried that floating point error is gonna push this value slightly over 1? Seems to me like we could just trim the value to 1 or -1

Reply:
I don't think floating point error is gonna push it over 1 but I'll trim anyway, it does sound like a better solution.

Reply:
Is there not a standard clamp function? line 354 is kinda hard to read lol

Reply:
ill try to find one!

Reply:
there is no standard clamp function but if u think it'll make it easier, i can turn it into a fx called clamp